tests/json/test_json.py

```diff
@@ -1,5 +1,5 @@
 """Test serialization and deserialization of gemd objects."""
-import json
+import json as json_builtin
```

Comment:
Why? I don't see anything else called `json`, and even if there was, I'd suggest aliasing the other thing rather than the built-in.